index.html

```diff
@@ -30,16 +47,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" id="b1">Box 1</div>
+                <div class="box" id="b2">Box 2</div>
+                <div class="box" id="b3">Box 3</div>
+                <div class="box" id="b4">Box 4</div>
+                <div class="box" id="b5">Box 5</div>
+                <div class="box" id="b6">Box 6</div>
+                <div class="box" id="b7">Box 7</div>
+                <div class="box" id="b8">Box 8</div>
+                <div class="box" id="b9">Box 9</div>
+                <div class="box" id="b10">Box 10</div>
```

Comment:
So this works, but I would use the id's sparingly. The was to do this in convention is to add another class. The reason being is because all of the boxes look like the class `box` and then you are only changing the color of the box. So the best way to do that is to add another class with a new color.

For Example

```<div class="box" id="b1">Box 1</div>```
Should be
```<div class="box box1">Box 1</div>```
And 
```<div class="box" id="b2">Box 2</div>```
Should be 
```<div class="box box2">Box 2</div>```

Notice how I just made another class instead of an id. Now in your css you can just style it like normal. 

Your way absolutely work! It's just as best practices, use another class to override styles on one class. You'll see why once we get to JavaScript 😉